index.html

```diff
@@ -3295,7 +3295,10 @@ <h3>Reserved Extension Points</h3>
               <td>`confidenceMethod`</td>
               <td>
 A property used for specifying one or more methods that a verifier 
-might rely on in order to authenticate the holder of a presentation. 
+might use to increase their confidence that the value of an attribute in or of
+a verifiable credential or verifiable presentation is accurate, including but not
+limited to attributes such `initialRecipient` (a/k/a `issuee`), `presenter`,
```

Comment:
I know I missed the merge. You can probably fix this with a direct commit, rather than going through all the PR hoops...
```suggestion
limited to attributes such as `initialRecipient` (a/k/a `issuee`), `presenter`,
```

Reply:
Thanks for catching this. Fixed in [cf429b1](https://github.com/w3c/vc-data-model/commit/cf429b194ff89ae255a70cf271aac45f703f96c1)